locustempus/main/tests/test_viewsets.py

```diff
@@ -259,6 +259,20 @@ class ResponseAPITest(CourseTestMixin, TestCase):
     def setUp(self):
         self.setup_course()
 
+    def create_student_response(self):
```

Comment:
Should this be `test_create_student_response`. Or is this a helper function?

Reply:
It should be `test_` Its now fixed.